can_tools/scrapers/official/IL/il_vaccine.py

```diff
@@ -11,88 +10,52 @@ class IllinoisVaccineCounty(StateDashboard):
     source = "https://www.dph.illinois.gov/covid19/vaccinedata"
     source_name = "Illinois Department of Public Health"
     state_fips = int(us.states.lookup("Illinois").fips)
-    url = "https://idph.illinois.gov/DPHPublicInformation/api/covidVaccine/getVaccineAdministrationCurrent"
+    url = "https://idph.illinois.gov/DPHPublicInformation/api/COVIDVaccine/getCOVIDVaccineAdministrationCountyAge"
     location_type = "county"
 
-    def fetch(self) -> dict:
-        res = requests.get(self.url)
-        if not res.ok:
-            msg = f"Could not request data from {self.url}"
-            raise ValueError(msg)
+    variables = {
+        "AdministeredCount": variables.INITIATING_VACCINATIONS_ALL,
```

Comment:
Are we sure this is correct?  "AdministeredCount" sounds like it'd be TOTAL_DOSES_ADMINISTERED_ALL, no?  If we're sure this is right, perhaps add a comment?

Reply:
It appears to be, although I don't see a way to truly verify it. There is a `TotalAdministered` column which looks like `total_vaccine_doses_administered`. The `AdministeredCount` seems like it lines up with `total_vaccine_initiated`, as it is never less than the `PersonsFullyVaccinated` column, and roughly lines up with our estimated 1+ dose (although is usually lower, often by a fair margin). 

I can add a comment to clarify this if we feel comfortable with shipping

Reply:
👍   Sounds good.  Let's ship it!